### ISSUE TITLE
GNS3 doesnt start in ubuntu 19.04.

### ISSUE THREAD

**Issue by sliddjur** [GNS3/gns3-gui#2774]
**Describe the bug**
GNS3 doesnt start in ubuntu 19.04

**GNS3 version and operating system (please complete the following information):**
 - OS: Ubuntu
 - GNS3 version: 2.1.16

**To Reproduce**
Install ubuntu 18.10 with latest gns3. Upgrade to ubuntu 19.04.
Try to start GNS3 from startmenu shortcut, or from terminal -> gns3.

**Screenshots or videos**

```
olof@olof-ubuntu ~> gns3
Could not find platform independent libraries <prefix>
Could not find platform dependent libraries <exec_prefix>
Consider setting $PYTHONHOME to <prefix>[:<exec_prefix>]
Fatal Python error: Py_Initialize: Unable to get the locale encoding
ModuleNotFoundError: No module named 'encodings'

Current thread 0x00007f588e5d4740 (most recent call first):
fish: “gns3” terminated by signal SIGABRT (Abort)
```

**Additional context**
Is no PPA package for disco (19.04) for ubuntu.

**Comment by adosztal** [comment]
Related to GNS3/gns3-server#1583.

**Comment by grossmj** [comment]
Let's see if this is fixed once 2.1.17 is released.

**Comment by grossmj** [comment]
This will be fixed when https://github.com/GNS3/gns3-server/issues/1583 is completed
